Automattic/wp-calypso
# Stats: Refactor layout styles and adjust for the Store page

#### Proposed Changes

* Refactor layout styles as a separate SCSS file.
* Adjust the layout styles on the Store page.

#### Testing Instructions



> Instructions to enable the `Store` page are mentioned in the [previous PR](https://github.com/Automattic/wp-calypso/pull/69661) or refer to the link `p1669058125021629-slack-C0438NHCLSY`

* Spin up this change with the Calypso Live link.
* Navigate to page `/stats/day/${your-site}`.
* Ensure the layout styles are the same as the previous one.
* Navigate to page `/store/stats/orders/day/${your-site}`.
* Ensure the layout styles are updated, like the `Traffic` and `Ads` pages.

| Before | After |
| --- | --- |
| <img width="1489" alt="store_before" src="https://user-images.githubusercontent.com/6869813/203138987-8d7f70a3-b7b6-4720-ac9d-36e8107b778a.png"> | <img width="1492" alt="store_after" src="https://user-images.githubusercontent.com/6869813/203138931-2dc03940-6579-4d9d-add4-abe3d002aacc.png"> |

#### Pre-merge Checklist





Related to #
